pkg/utils/helpers.go

```diff
@@ -159,7 +161,7 @@ func ConvertYamlFileToMap(fileName string) ([]map[string]interface{}, error) {
 	return retVal, nil
 }
 
-func ConvertYamlFileToArrayOfMaps(fileName string) ([]map[string]interface{}, error) {
+func (h *YamlConfigHandler) ToArrayOfMaps(fileName string) ([]map[string]interface{}, error) {
```

Comment:
rearranage this file to have the type next to its functions
or whatever order that you think its best
for readability puposes